### PR TITLE
update versions for Dec release

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Note, I have included the change to ForceCloseHandles under the assumption that will be making it into the release.

Also updated the Readme files to point to docs.microsoft.com rather than github (except for Blobs.Batch which is missing from docs.microsoft.com - following up on this).

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Release History
 ===============
+12.1.0
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md]: Nit - add a blank line above the version like the previous entries (and everywhere else)

File: sdk/storage/Azure.Storage.Files.DataLake/Changelog.txt [hunk 2]
```diff
@@ -1,7 +1,16 @@
 Release History
 ===============
+12.0.0-preview.7
+--------------------------
+- Added check to enforce TokenCredential is used only over HTTPS
+- Enabled diagnostic tracing
+- Added FileSystemClient.GetAccessPolicy and SetAccessPolicy
+- Added ToBlobUri/ToDfsUri methods to DataLakeUriBuilder
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Files.DataLake/Changelog.txt]: Aren't these internal?

File: sdk/storage/Azure.Storage.Files.Shares/BreakingChanges.txt [hunk 3]
```diff
@@ -1,5 +1,10 @@
 Breaking Changes
 ================
+12.0.0
+--------------------------
+- Updated ShareFileRangeInfo to use HttpRange
+- Update return types of ForceCloseAllHandles/ForceCloseHandle to be CloseHandlesInfo
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Files.Shares/BreakingChanges.txt]: Nit - `ClosedHandlesInfo`.  I had a typo in my version.

**Review by tg-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Files.DataLake/Changelog.txt (hunk 2 above)
Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Files.DataLake/Changelog.txt]: Removed

**Review by tg-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Files.DataLake/README.md [hunk 4]
```diff
@@ -233,7 +233,7 @@ additional questions or comments.
 <!-- LINKS -->
 [source]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Files.DataLake/src
 [package]: https://www.nuget.org/packages/Azure.Storage.Files.DataLake/
-[docs]: https://azure.github.io/azure-sdk-for-net/storage.html
+[docs]: https://docs.microsoft.com/en-us/dotnet/api/azure.storage.files.datalake?view=azure-dotnet-preview
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Files.DataLake/README.md]: Can we drop the `view=` query params on these so we don't have to update them in the future?  I think it'll default to latest if not provided.

**Review by tg-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Files.DataLake/README.md (hunk 4 above)
Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Files.DataLake/README.md]: Updated

**Review by seanmcc-msft — APPROVED** [review]

**Pull request merged**